CMakeLists.txt

```diff
@@ -58,45 +59,53 @@ CATKIN_DEPENDS message_runtime geometry_msgs std_msgs
 ## Your package locations should be listed before other locations
 # include_directories(include)
 include_directories(
+  src
   ${catkin_INCLUDE_DIRS}
 )
 
 #ratslam library
 add_library(ratslam src/ratslam/experience_map.cpp src/ratslam/posecell_network.cpp src/ratslam/local_view_match.cpp src/ratslam/visual_odometry.cpp)
 
+target_link_libraries(ratslam
+  ${Boost_LIBRARIES}
+)
```

Comment:
That's interesting as the travis was able to build it in this configuration. I will check this!